genrules/BUILD

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 EngFlow Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# NOTE
+# The following genrules allow for an intensive test on the building system
+# as they create thousands of targets to be specified in few lines of code.
+# WARNING: Be careful with the number of targets created as it may be
```

Comment:
Remove the next comments, they were used for testing but are not very useful outside of that context.